docs/topics/native/native-get-started.md

```diff
@@ -23,22 +23,43 @@ To get started, install the latest version of [IntelliJ IDEA](https://www.jetbra
 
 Your project will open. By default, the wizard creates the necessary `main.kt` file with code that prints "Hello, Kotlin/Native!" to the standard output.
 
-The `build.gradle.kts` file contains the project settings. Read more about these settings in the [Kotlin Multiplatform Gradle DSL reference](mpp-dsl-reference.md).
+The `build.gradle.kts` file contains the project settings. Make sure you use the latest version of the Kotlin Multiplatform
+Gradle plugin:
 
-## Run the application
+```kotlin
+plugins {
+    kotlin("multiplatform") version "%kotlinVersion%"
+}
+```
+   
+Read more about these settings in the [Kotlin Multiplatform Gradle DSL reference](mpp-dsl-reference.md).
 
-Start the application by clicking **Run** next to the run configuration at the top of the screen.
+## Build and run the application
 
-![Run the application](native-run-app.png){width=600}
+1. Click **Build Project** next to the run configuration at the top of the screen:
 
-IntelliJ IDEA opens the **Run** tab and shows the output:
-![Application output](native-output-1.png){width=600}
+   ![Build the application](native-run-app.png){width=600}
+
+2. On the **Terminal** tab, run the following command:
+
+   ```bash
+   build/bin/native/debugExecutable/<your_app_name>.kexe
+   ```
+
+   IntelliJ IDEA prints "Hello, Kotlin/Native!".
+
+You can [configure IntelliJ IDEA](https://www.jetbrains.com/help/idea/compiling-applications.html#auto-build) to build
+your project automatically every time you make changes:
+
+1. Go to **Settings/Preferences | Build, Execution, Deployment | Compiler**.
+2. On the **Compiler** page, select **Build project automatically**.
+3. Apply the changes.
 
 ## Update the application
 
 ### Count the letters in your name
 
-1. Open the file `main.kt` in `src/<your_app_name>Main/kotlin`.
+1. Open the file `main.kt` in `src/nativeMain/kotlin`.
```

Comment:
Well, we didn't provide any specific name before...
Are you sure about this change?

Reply:
Well, nativeMain is the default name. We don't advise to change it in the first instruction, step 4 ("accept the default configuration")

Reply:
Don't it correlate with the project/app name?